src/components/atoms/Sticky.tsx

```diff
@@ -0,0 +1,82 @@
+import { css } from '@emotion/react'
+import { ReactNode, useCallback, useEffect, useRef, useState } from 'react'
+
+export const getScrollTop = () => {
+  if (!document.body) return 0
+
+  return document.documentElement
+    ? document.documentElement.scrollTop || document.body.scrollTop
+    : document.body.scrollTop
+}
+
+const stickyBlock = css`
+  width: 100%;
+`
+
+export interface StickyProps {
+  bottom: number
+  className?: string
+  children: ReactNode
+}
+
+function Sticky({ className, bottom, children }: StickyProps) {
+  const [y, setY] = useState(0)
+  const element = useRef<HTMLDivElement | null>(null)
```

Comment:
```suggestion
  const element = useRef<HTMLDivElement>(null)
```
`useRef<T = undefined>()` 으로 제너릭을 받고 있어서 따로 위 처럼 작성하면 좀 더 깔끔하지 않을까요?!

Reply:
function useRef<T>(initialValue: T|null): RefObject<T>; 이네요 
좋은 지적 감사합니다. 🙏🏻

Reply:
https://github.com/Lubycon/clelab-frontend/pull/9/commits/46430937d8f665719ea89f4b0864b8f0c196c8db 해당 반영하였습니다.